google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -209,6 +210,12 @@ public CloudStoragePath getPath(URI uri) {
         getFileSystem(CloudStorageUtil.stripPathFromUri(uri)), uri.getPath());
   }
 
+  /** Convenience method: replaces spaces with "%20", builds a URI, and calls getPath(uri). */
+  public CloudStoragePath getPath(String uriInStringForm) {
+    String escaped = UrlEscapers.urlFragmentEscaper().escape("gs://bucket/with/a space");
```

Comment:
I think you want `uriInStringForm` here instead of `"gs://bucket/with/a space"` :)